qmlti/student_nav.php

```diff
@@ -0,0 +1,137 @@
+<?php
```

Comment:
Do you really want to mix all this business logic in with the presentation logic? Perhaps you could extract this into a separate class and simulate MVC slightly.